radis/levels/partfunc.py

```diff
@@ -1170,6 +1170,8 @@ def __init__(self,species):
 
     def _at(self, T):
         # Interpolate to find the partition function at the desired temperature
+        if T<10**(-5) or T>10**4:
```

Comment:
- You can just put `>= 0` for the minimum bound because I think the partition functions are evaluated from T = 0 K. You have to check.

- Why did you put 10,000 K as the maximum? Is it the same for all molecules?

Reply:
I put these lower and upper bounds because according to the values I have from Exojax, the partition functions are evaluated at these temperatures : 
pfT_values = np.array(
            [
                1e-05,
                1e-04,
                1e-03,
                1e-02,
                1e-01,
                1.5e-01,
                2e-01,
                3e-01,
                5e-01,
                7e-01,
                1.0,
                1.3,
                1.7,
                2.0,
                3.0,
                5.0,
                7.0,
                10.0,
                15.0,
                20.0,
                30.0,
                50.0,
                70.0,
                100.0,
                130.0,
                170.0,
                200.0,
                250.0,
                300.0,
                500.0,
                700.0,
                1e03,
                1.5e03,
                2e03,
                3e03,
                4e03,
                5e03,
                6e03,
                7e03,
                8e03,
                9e03,
                1e04,
            ])
            @minouHub

Reply:
I cannot find easily an upper bound for the validity of these partition functions. Users can easily go past 10,000 K in their simulations (during my thesis I went up to 50,000 K in my experiments). Please increase the max up to 50,000 K and add a `UserWarning` for now when the partition functions are evaluated above 20,000 K. It could be `The validity of the Kurucz partition function is not checked for now. Please be careful in interpretations.`

Reply:
@minouHub by default I'd rather raise an error (as it is now) than extrapolate. Partition functions might change a lot in between 10,000 K and 50,000 K.  

If you want to compute above 10,000 K ; maybe Kurucz partition functions aren't the right choice. 
For higher temperature spectra users would have to provide their own partition functions.  How to do that is not straighforward (can be done by editing the radis.json database), but making it more clear & adding an example can be work for later!